adam-core/src/main/scala/org/bdgenomics/adam/models/SingleReadBucket.scala

```diff
@@ -21,15 +21,14 @@ import com.esotericsoftware.kryo.{ Kryo, Serializer }
 import com.esotericsoftware.kryo.io.{ Output, Input }
 import org.apache.spark.Logging
 import org.apache.spark.rdd.RDD
+import org.bdgenomics.adam.rdd.ADAMContext
 import org.bdgenomics.adam.rdd.ADAMContext._
 import org.bdgenomics.adam.serialization.AvroSerializer
-import org.bdgenomics.formats.avro.{
-  AlignmentRecord,
-  Fragment
-}
+import org.bdgenomics.formats.avro.{ SingleReadBucket, AlignmentRecord, Fragment }
+
 import scala.collection.JavaConversions._
 
-object SingleReadBucket extends Logging {
+object SingleReadBucketRDD extends Logging {
```

Comment:
Shouldn't this move to the `org.bdgenomics.adam.rdd.read` package and become package private?